src/data_handler.py

```diff
@@ -0,0 +1,160 @@
+#import BoardInterface
+
+from enum import Enum
+
+class Sensor():
+
+    def __init__(self, sensorID, boardID, name):
+        self.sensorID = sensorID
```

Comment:
It's worth documenting what ranges of values are expected for the sensor and board IDs - i.e. how many bits each are, and what order they appear in the message (I'd have to check, but iirc, its board then sensor)